ftw/noticeboard/configure.zcml

```diff
@@ -28,4 +29,8 @@
         directory="upgrades"
         />
 
+    <include file="permission.zcml" />
+    <include zcml:condition="installed ftw.lawgiver" file="lawgiver.zcml" />
```

Comment:
Was wird damit bezweckt?

Reply:
Damit wird alles was mit den Berechtigungen/ftw.lawgiver in eigene Dateien ausgelagert. Das wir so in vielen ftw.* Packages gemacht. So zum Beispiel auch bei ftw.news -> https://github.com/4teamwork/ftw.news/blob/master/ftw/news/configure.zcml#L18-L19

Reply:
Sehr verständlich, danke für die Ausfühung!